### PR TITLE
First WARG Commit

### PR DESCRIPTION
**Pull request by noah-yacowar**
The first screenshot includes console output of accuracies. The training dataset accuracy was just under 96% accurate, and same with the testing dataset. 

In the second picture the second graph includes both the training data set and validation dataset losses. 

Please let me know what else is needed!

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,17 +1,271 @@
...
+
+# Creating dataloading wrappers for the 3 datasets, used to access data more efficiently
+batch_size = 32
+train_dl = torch.utils.data.DataLoader(train_dataset, batch_size, shuffle=True, pin_memory=True)
+val_dl = torch.utils.data.DataLoader(val_dataset, batch_size, shuffle=True, pin_memory=True)
+test_dl = torch.utils.data.DataLoader(test_dataset, batch_size, pin_memory=True)
+
+
+# Determines whether the machine is Cuda enabled, allowing GPU access
+def get_default_device():
```
Comment (mgupta27) [inline main.py:50]: Nit: It is good practice to also specify what the return type will be for the function. It helps make the code more readable.

**Review by mgupta27 — APPROVED** [review]
LGTM!

Congratulations and welcome to the Autonomy sub-team at WARG. For next steps please refer to https://uwarg-docs.atlassian.net/wiki/spaces/AD/pages/1697349760/New+Member+Onboarding. You will notice there is also a subpage for new Autonomy members as well. Please take a look at that as well.

Meetings are TBD but will be announced soon in the Discord. Can't wait to see you there!

File: main.py [hunk 2]
```diff
@@ -1,17 +1,271 @@
 """
```
Comment (mgupta27) [inline main.py:1]: Note: Regarding naming convention for PR's, it's good practice to name the PR relevant to the task you are working on. A PR is different from a commit so "First WARG Commit" isn't a great a PR name (but that is ok because we are here to learn and it is totally fine to make mistakes. I've made some myself when I first joined 😄). If you would like more help with understanding Git feel free to reach out.
